README.md

```diff
@@ -558,6 +558,74 @@ With the configuration above, when you run `terragrunt apply`, Terragrunt will c
 terraform apply -lock-timeout=20m -var-file=terraform.tfvars -var-file=terraform-secret.tfvars
 ```
 
+#### Required and conditional var-files
+
+Sometime, it is required to include variables to terraform conditionally. The problem is that if we use
+-var-file=<file.tfvars>, <file.tfvars> must have to exist, otherwise, we get an error from terraform.
+
+For example, you may want to include files if they are available only.
```

Comment:
Also, the examples above this one should be changed to NOT use `-var-file` but `-var` or some other command line param, as `-var-file` should be handled using these helpers.